<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume Rendering</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(30, 30, 50);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="slider_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Iso Value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="load_label">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>180</y>
      <width>711</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Loaded Directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QSlider" name="ScalarOpacity_slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="GradientOpacity_slider">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ScalarOpacity_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Scalar Opacity</string>
    </property>
   </widget>
   <widget class="QLabel" name="GradientOpacity_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gradient Opacity</string>
    </property>
   </widget>
   <widget class="QLabel" name="ScalarOpacity_val">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 8pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="GradientOpacity_val">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 8pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="colors_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Colors</string>
    </property>
   </widget>
   <widget class="QComboBox" name="renMode_CB">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 127);</string>
    </property>
    <item>
     <property name="text">
      <string>Ray Casting</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Surface Rendering</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="iso_slider">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>41</y>
      <width>761</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>2500</number>
    </property>
    <property name="singleStep">
     <number>50</number>
    </property>
    <property name="pageStep">
     <number>50</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>19</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 8pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>500</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(67, 67, 75);</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenImgs">
   <property name="text">
    <string>Open Images</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_new_window">
   <property name="text">
    <string>New Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPrint_to_PDF">
   <property name="text">
    <string>Print to PDF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionAuto_Save">
   <property name="text">
    <string>Auto Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpenImg2">
   <property name="text">
    <string>Open Img2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionopen_both">
   <property name="text">
    <string>open both</string>
   </property>
  </action>
  <action name="close">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>iso_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>730</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ScalarOpacity_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ScalarOpacity_val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GradientOpacity_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GradientOpacity_val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
